tspserver/src/main/java/com/komiwojazer/bean/User.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
We wszystkich edytorach jakie mamy bym wyłączył automatyczne dodawanie liencji i autorów, bo każdy edytor będzie swoje dodawał. Autora danego fragmentu kodu zawsze można znaleźć przy pomocy `git blame`.

Reply:
zrobione